hugr-py/tests/test_hugr_build.py

```diff
@@ -0,0 +1,177 @@
+import subprocess
+
+from hugr.hugr import Dfg, Hugr, DummyOp, Node
+import hugr.serialization.tys as stys
+import hugr.serialization.ops as sops
+import pytest
+
+BOOL_T = stys.Type(stys.SumType(stys.UnitSum(size=2)))
+QB_T = stys.Type(stys.Qubit())
+ARG_5 = stys.TypeArg(stys.BoundedNatArg(n=5))
+INT_T = stys.Type(
+    stys.Opaque(
+        extension="arithmetic.int.types",
+        id="int",
+        args=[ARG_5],
+        bound=stys.TypeBound.Eq,
+    )
+)
+
+NOT_OP = DummyOp(
+    # TODO get from YAML
+    sops.CustomOp(
+        parent=-1,
+        extension="logic",
+        op_name="Not",
+        signature=stys.FunctionType(input=[BOOL_T], output=[BOOL_T]),
+    )
+)
+
+DIV_OP = DummyOp(
+    sops.CustomOp(
+        parent=-1,
+        extension="arithmetic.int",
+        op_name="idivmod_u",
+        signature=stys.FunctionType(input=[INT_T] * 2, output=[INT_T] * 2),
+        args=[ARG_5, ARG_5],
+    )
+)
+
+
+def _validate(h: Hugr, mermaid: bool = False):
+    # cmd = ["cargo", "run", "--features", "cli", "--"]
+    cmd = ["./target/debug/hugr"]
```

Comment:
This assumes we've built a debug version of our rust tool in this same environment... I see that failing unexpectedly.

Can't we require `hugr` to either be available on PATH, or be defined in an env var?

Reply:
yeah sorry this is local testing state, I like the env var option